UI/src/utils/backend-utils.js

```diff
@@ -487,10 +483,6 @@
       return $.getJSON('api/suggested/' + municipalityCode + '/' + typeId);
     };
 
-    this.getStructuredInfo = function(municipalityName, municipalityCode) {
-      return $.getJSON('api/buildInfo/' + municipalityName + '/' + municipalityCode);
-    };
-
```

Comment:
Are this removes ok?

Reply:
Attention on this when merge master inside

Reply:
getStructuredInfo is not being used so the remove can be applied